services/validatorWorker/leader.js

```diff
@@ -17,7 +20,15 @@ function afterProducer(adapter, {channel, newStateTree, balances}) {
 		acc => adapter.getBalanceLeaf(acc, balances[acc])
 	)
 	const tree = new adapter.MerkleTree(elems)
-	const stateRootRaw = tree.getRoot()
+	const balanceRoot = tree.getRoot()
```

Comment:
rather than importing `keccak256`, why don't we just add to the adapter `adapter.getSignableRoot` that would be invoked like `adapter.getSignableRoot(channel, tree.getRoot())` 

this keeps the follower/leader code generic, and allows us to later add more stuff from `channel` into the signable root

Reply:
ok making the change